eclipse-cdt-cloud/cdt-cloud-blueprint
# Update to Blueprint and Theia 1.30

#### What it does
Update CDT.cloud Blueprint to the latest Theia Blueprint and therefore Theia 1.30

Also updates to the latest prerelease of TraceCompass.cloud

Contributed on behalf of STMicroelectronics.

#### How to test
Build and run CDT.cloud Blueprint in all variants and check whether all examples work as expected
